lv_objx/lv_chart.c

```diff
@@ -819,4 +821,70 @@ static void lv_chart_draw_vertical_lines(lv_obj_t * chart, const lv_area_t * mas
         }
     }
 }
+
+/**
+ * Draw the data lines as areas on a chart
+ * @param obj pointer to chart object
+ */
+static void lv_chart_draw_areas(lv_obj_t * chart, const lv_area_t * mask)
+{
+	lv_chart_ext_t * ext = lv_obj_get_ext_attr(chart);
+
+	uint16_t i;
+	lv_point_t p1;
+	lv_point_t p2;
+	lv_coord_t w = lv_obj_get_width(chart);
+	lv_coord_t h = lv_obj_get_height(chart);
+	lv_coord_t x_ofs = chart->coords.x1;
+	lv_coord_t y_ofs = chart->coords.y1;
+	int32_t y_tmp;
+	lv_coord_t p_prev;
+	lv_coord_t p_act;
+	lv_chart_series_t * ser;
+	lv_opa_t opa_scale = lv_obj_get_opa_scale(chart);
+	lv_style_t style;
+	lv_style_copy(&style, &lv_style_plain);
+	style.line.opa = ext->series.opa;
+	style.line.width = ext->series.width;
+
+	/*Go through all data lines*/
+	LL_READ_BACK(ext->series_ll, ser) {
```

Comment:
@ali-rostami This needs to be changed to `LV_LL_READ_BACK` because it needs to be merged to `v6.0` not `v5.3`.

Reply:
thanks, @embeddedt. I changed it.